### ISSUE TITLE
Inverse-squared link error

### ISSUE THREAD

**Issue by loicguille** [theislab/cellrank#1170]
Hello,
Thanks for the amazing work,
I am trying to create a model : 
```python
modelGamaInv = cr.models.GAM(SL2_ddc, distribution = "inv_gauss", link = "inverse-squared",n_knots = 10)
```
When I try to use this model I face an error with CellRank  : 

```python
cr.pl.gene_trends(
    SL2_ddc,
    model=modelGamaInv,
    lineages="BEC_Adult",
    data_key="magic_imputed_data",
    genes=gene_list,
    same_plot=True,
    ncols=2,
    time_key="slingshot_lineage2",
    cell_color="new_labelling",
    hide_cells=False,
    weight_threshold=(1e-3, 1e-3),
    size =5
)
```
```
 Traceback (most recent call last):
  File "/miniconda3/lib/python3.9/site-packages/cellrank/models/_pygam_model.py", line 203, in fit
    self.model.fit(self.x, self.y, weights=self.w, **kwargs)
  File "/miniconda3/lib/python3.9/site-packages/pygam/pygam.py", line 886, in fit
    self._validate_params()
  File "/miniconda3/lib/python3.9/site-packages/pygam/pygam.py", line 265, in _validate_params
    raise ValueError('unsupported link {}'.format(self.link))
ValueError: unsupported link inverse-squared
```
Have you an idea of what can lead to this error ?

I use CellRank version 2.0.0 and pygam version 0.8.0

Thanks

Loïc

**Comment by Marius1311** [comment]
@michalk8 might know - I don't think we have tested this extensively with different link functions.

**Comment by michalk8** [comment]
Hi @loicguille , seems like there's a typo in the docs, can you try `link="inv_squared"`?

**Comment by loicguille** [comment]
Hi, when I use the command you suggest : 
```python
modelSL2 = cr.models.GAM(SL2_ddc, distribution = "inv_gauss", link = "inv_squared",n_knots = 10)
```
I get another error : 
```
ValueError: Invalid option `'inv_squared'` for `GamLinkFunction`. Valid options are: `['identity', 'logit', 'inverse', 'log', 'inverse-squared']`.
```
Thanks for the reply.

**Comment by WeilerP** [comment]
@loicguille, can you please install CellRank from `fix/gam_link_names` and try again? See #1175.

**Comment by loicguille** [comment]
@WeilerP, It 's working perfectly fine now, thanks for your time. 

Loïc
